packages/web-ui/package.json

```diff
@@ -73,8 +75,8 @@
     "lucide-react": "^0.468.0",
     "monaco-editor": "^0.50.0",
     "postcss": "^8.4.39",
-    "promptl-ai": "^0.5.2",
     "postcss-import": "^16.1.0",
+    "promptl-ai": "^0.5.2",
     "react": "18.3.0",
     "react-dom": "18.3.0",
     "react-resizable": "^3.0.5",
```

Comment:
Uhmm, I don't see `Command` we were already using it?

Reply:
it's in atoms/Command